docs/index.md

```diff
@@ -94,9 +94,9 @@ you can install via pip, conda-forge, or from source.
 
 ### From pip, with "batteries included"
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
-3.8-3.10 using pip:
-
+napari can be installed on most macOS, Linux, and Windows systems with
+ ![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg) by
```

Comment:
I personally don't like the shields in-line with text, but it's not a blocker.
<img width="731" alt="image" src="https://user-images.githubusercontent.com/76622105/209573600-33b3cad9-1114-4a22-96ef-6e7e8727391f.png">